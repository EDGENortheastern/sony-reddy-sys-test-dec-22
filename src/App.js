import logo from './images/logopic.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <h1>
          This is the page for dog lovers
        </h1>
      </header>
    </div>
  );
}

export default App;