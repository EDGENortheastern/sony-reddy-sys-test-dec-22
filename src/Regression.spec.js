import renderer from 'react-test-renderer';

import App from './App';

it('checks that App Dom renders correctly', () => {

    const appTree = renderer.create(<App/>).toJSON();

    expect(appTree).toMatchSnapshot();

}) 