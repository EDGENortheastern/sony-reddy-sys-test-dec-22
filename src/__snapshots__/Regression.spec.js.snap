// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks that App Dom renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logopic.svg"
    />
    <h1>
      When a number has been multiplied by itself
    </h1>
    <h2
      className="greeting-h2"
    >
      Type a number to find the square
    </h2>
    <input
      id="input-box"
      onChange={[Function]}
      type="number"
      value=""
    />
    <p>
      Result 
      0
    </p>
    <button
      onClick={[Function]}
    >
      Clear
    </button>
  </header>
</div>
`;
